code/src/main/kotlin/com/expediagroup/sdk/core/common/Feature.kt

```diff
@@ -0,0 +1,5 @@
+package com.expediagroup.sdk.core.common
```

Comment:
Please add docs stings and license headers

Reply:
done.